tracing-tower/src/lib.rs

```diff
@@ -71,33 +71,8 @@ where
     }
 }
 
-#[cfg(feature = "tower-util")]
```

Comment:
These could be migrated to `tower_make` instead, see #2971

Reply:
Yeah, see the commit description for why I didn't do that.